week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/shop/cart-item.component.ts

```diff
@@ -0,0 +1,17 @@
+import { div } from '../framework/dom-creators.js'
+import { Item } from '../model/Item'
+
+export function cartItem({ name, amount, unit, price }: Item) {
```

Comment:
Tutaj też `:HTMLDivElement ` skoro w `cart-item-sum.component` to było, to tutaj też dodaj zwracany przez funkcję typ

Reply:
Dodane